lib/email_reply_parser.rb

```diff
@@ -50,6 +70,18 @@ def self.parse_reply(text)
     self.read(text).visible_text
   end
 
+  ### Configuration
+
+  # A Configuration instance.
+  class Configuration
+    # Configuration has an Array of regards
+    attr_accessor :regards
```

Comment:
I think, at least generally in English, we'd refer to:

>Warm regards,
> Matt

as a signature as well, just a particular type of signature.

Rather than introduce the more-specific domain term `regards`, I think we should name this in terms of `signature` customization.